src/raftstore/store/fsm/apply.rs

```diff
@@ -2766,11 +2769,7 @@ impl PollHandler<ApplyFsm, ControlFsm> for ApplyPoller {
             }
         }
         normal.handle_tasks(&mut self.apply_ctx, &mut self.msg_buf);
-        if normal.delegate.merged {
```

Comment:
seems it's never updated? legacy code? @BusyJay

Reply:
Yes. It's legacy code.